pkg/brctl/brctl.go

```diff
@@ -0,0 +1,325 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package brctl
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+)
+
+func Addbr(name string) error {
+	brctl_socket, err := unix.Socket(unix.AF_INET, unix.SOCK_STREAM, 0)
```

Comment:
Not sure if this is correct name, in general variables names should not contain underscores. See https://google.github.io/styleguide/go/decisions#underscores It's ok to reuse them, but probably brctlSocket will preferable here.

Reply:
Agree.